src/ActionBar/ActionBar.js

```diff
@@ -1,31 +1,39 @@
-import 'fundamental-styles/dist/action-bar.css';
 import ActionBarActions from './_ActionBarActions';
 import ActionBarBack from './_ActionBarBack';
 import ActionBarHeader from './_ActionBarHeader';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
+import withStyles from '../utils/WithStyles/WithStyles';
 
-const ActionBar = ({ children, className, ...props }) => {
+const ActionBar = React.forwardRef(({ children, className, customStyles, disableStyles, ...props }, ref) => {
     const actionBarClasses = classnames(
         'fd-action-bar',
         className
     );
 
     return (
-        <div {...props} className={actionBarClasses}>{children}</div>
+        <div {...props}
+            className={actionBarClasses}
+            ref={ref}>
```

Comment:
The reasoning was to go ahead and add them to every functional component being wrapped in `withStyles` as it will be a breaking change to add them going forward. See: https://reactjs.org/docs/forwarding-refs.html#note-for-component-library-maintainers